autosubmit/autosubmit.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+allow_config_override: false
+auto_approval_accounts:
+  - dependabot[bot]
+  - dependabot
+approving_reviews: 2
+approval_group: dart-team
+run_ci: true
+support_no_review_revert: true
+required_checkruns:
+  - "cla/google"
+  - "analyze (dev)"
```

Comment:
Seeing this now - I assume that this current list is here while we build out the feature, but we won't need to list specific checkrun names in the future? There are many, many permutations of job names on our repos.

Reply:
Sure you can leave this blank as it only applies to revert. When the pull requests run it will make sure all check runs/github actions have finished.

Reply:
Great!